en/README.md

```diff
@@ -0,0 +1,1138 @@
+  },
+  "method": "putArmInfo",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Receive result as result object value whether sent data has been updated. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchItems(method)
+
+This procedure request to get item to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched items using a putItems procedure. At that time, need to pass fetchId and hostIds of fetchItems procedure params to putItems procedure.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostIds|String255 array|O|-|Get triggers of to designate a hosts. If does not designate hostIds, all trigger are targeted.|
+|fetchId|String255    |M|-|Use in putItems procedures. It need to identify the putItems corresponding to the fetchItems.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "fetchId": "1",
+    "hostIds": [
+      "1",
+      "2",
+      "3"
+    ]
+  },
+  "method": "fetchItems",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Return result as result object value whether request accept to Hatohol server. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchHistory(method)
+
+This procedure request to get history to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched history using a putHistory procedure. At that time, need to pass fetchId of fetchHistory procedure params to putHistory procedure.
+
+Assume to get matched history that from beginTime till endTime.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostId    |String255|M|-|Host Id that the item belongs.|
+|itemId    |String255|M|-|Item Id of history.|
+|beginTime |TimeStamp|M|-|Starting point of history.|
+|endTime   |TimeStamp|M|-|End point of history.|
+|fetchId|String255    |M|-|Use in putHistory procedures. It need to identify the putHistory corresponding to the fetchHistory.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "fetchId": 1,
+    "beginTime": "20150323151300",
+    "valueType": "INTERGER",
+    "itemId": 1,
+    "hostId": "1"
+  },
+  "method": "fetchHistory",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Return result as result object value whether request accept to Hatohol server. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchTriggers(method)
+
+This procedure request to get triggers to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched triggers using a putTriggers procedure. At that time, need to pass fetchId and hostIds of fetchTriggers procedure params to putTriggers procedure.
```

Comment:
> At that time, need to pass fetchId and hostIds of fetchTriggers procedure params to putTriggers procedure.

And then, it need**s** to pass fetchId which is contained in fetchTriggers procedure paramemeters to putTriggers procedure.